PokeApiNet/Models/Pokemon.cs

```diff
@@ -785,6 +785,988 @@ public class PokemonSprites
+                    [JsonProperty("back_shiny")]
+                    public string BackShiny { get; set; }
+
+                    /// <summary>
+                    /// The default depiction of this Pokémon from the front in battle.
+                    /// </summary>
+                    [JsonProperty("front_default")]
+                    public string FrontDefault { get; set; }
+
+                    /// <summary>
+                    /// The default depiction of this Pokémon from the front shiny in battle.
+                    /// </summary>
+                    [JsonProperty("front_shiny")]
+                    public string FrontShiny { get; set; }
+
+                    /// <summary>
+                    /// The default depiction of this Pokémon from the front in battle on transparent background.
+                    /// </summary>
+                    [JsonProperty("front_transparent")]
+                    public string FrontTransparent { get; set; }
+                }
+
+                /// <summary>
+                /// Pókemon Silver sprites
+                /// </summary>
+                public class SilverSprites
+                {
+                    /// <summary>
+                    /// The default depiction of this Pokémon from the back in battle.
+                    /// </summary>
+                    [JsonProperty("back_default")]
+                    public string BackDefault { get; set; }
+
+                    /// <summary>
+                    /// The default depiction of this Pokémon from the back shiny in battle.
+                    /// </summary>
+                    [JsonProperty("back_shiny")]
+                    public string BackShiny { get; set; }
+
+                    /// <summary>
+                    /// The default depiction of this Pokémon from the front in battle.
+                    /// </summary>
+                    [JsonProperty("front_default")]
+                    public string FrontDefault { get; set; }
+
+                    /// <summary>
+                    /// The default depiction of this Pokémon from the front shiny in battle.
+                    /// </summary>
+                    [JsonProperty("front_shiny")]
+                    public string FrontShiny { get; set; }
+
+                    /// <summary>
+                    /// The default depiction of this Pokémon from the front in battle on transparent background.
+                    /// </summary>
+                    [JsonProperty("front_transparent")]
+                    public string FrontTransparent { get; set; }
+                }
+            }
+
+            /// <summary>
+            /// Pókemon sprites for Generation III
+            /// </summary>
+            public class GenerationIIISprites
+            {
+                /// <summary>
+                /// Pókemon Emerald sprites
+                /// </summary>
+                [JsonProperty("emerald")]
+                public EsmeraldSprites Emerald { get; set; }
```

Comment:
Let's change the class from `EsmeraldSprites` to `EmeraldSprites`, please.

Reply:
Oh, man. I thought I had corrected them all. Sorry! Done.